CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.7.0](https://github.com/sonata-project/SonataUserBundle/compare/4.6.0...4.7.0) - 2020-07-27
+### Changed
+- [[#1183](https://github.com/sonata-project/SonataUserBundle/pull/1183)]
+  SonataEasyExtendsBundle is now optional, using SonataDoctrineBundle is
+preferred ([@jordisala1991](https://github.com/jordisala1991))
+- [[#1196](https://github.com/sonata-project/SonataUserBundle/pull/1196)]
```

Comment:
IMO, this one should be considered pedantic, since this should not be perceived by final users.

Reply:
Then why did you tag it at minor :sweat_smile: ?

Reply:
That being said I agree with you, let's drop it.

Reply:
My bad :facepalm: